scene/resources/sky_material.cpp

```diff
@@ -308,12 +308,23 @@ Ref<Texture2D> PanoramaSkyMaterial::get_panorama() const {
 	return panorama;
 }
 
+void PanoramaSkyMaterial::set_filtering_enabled(bool p_enabled) {
+	filter = p_enabled;
+	RS::get_singleton()->material_set_param(_get_material(), "filter", filter);
```

Comment:
I don't see "filter" as a uniform in your shader, so this is likely updating nothing